src/figuresClasses.ts

```diff
@@ -1,19 +1,81 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  color: Color,
+  getArea(): number,
+  shape?: Shape,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape: Shape = 'triangle';
 
-}
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Wrong data');
```

Comment:
Please re-check 4 poin in checklist and provide more representive error. Please fix for all cases
![image](https://github.com/mate-academy/ts_figures_classes/assets/54936044/e5d7ac25-87e2-4a58-95bb-cf6bf098c9f7)